File: Laravel/app/Models/ProdukModel.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class ProdukModel extends Model
{
    use HasFactory;
    protected $table = 'tb_produk';
 
    protected $fillable = [ 
        'id_franchisor',
        'id_kategori',
        'nama', 
        'harga',
        'gambar', 
    ];
}

File: Android/app/src/main/java/com/appfranchisor/app/ui/franchisee/model/ProdukModel.kt
package com.appfranchisor.app.ui.franchisee.model

import com.google.gson.annotations.SerializedName

data class ProdukModel(
    @SerializedName("id"            ) var id           : Int?    = null,
    @SerializedName("id_franchisor" ) var idFranchisor : Int?    = null,
    @SerializedName("id_kategori"   ) var idKategori   : Int?    = null,
    @SerializedName("nama"          ) var nama         : String? = null,
    @SerializedName("harga"         ) var harga        : Int?    = null,
    @SerializedName("gambar"        ) var gambar       : String? = null,
    @SerializedName("created_at"    ) var createdAt    : String? = null,
    @SerializedName("updated_at"    ) var updatedAt    : String? = null)
